### ISSUE TITLE
Error occurred while running twitterStream.py "ImportError: No module named 'PyQt4' "

### ISSUE THREAD

**Issue by aniketp3012** [sridharswamy/Twitter-Sentiment-Analysis-Using-Spark-Streaming-And-Kafka#1]
When I am running twitterStream.py following error occurs:
ImportError: No module named 'PyQt4'
I tried to install 'PyQt4' using pip install but it says that 'No matching distribution found for PyQt4'
Kindly help

**Comment by saurabhbidwai** [comment]
Try this
sudo yum install PyQt4

**Comment by jerchinn** [comment]
Is this code working fine...I could see many users posted their errors here.

**Comment by saurabhbidwai** [comment]
Nope, u ll face a lot of errors while running the code, need to do a lot of changes
